### ISSUE TITLE
Unhandled error if cookies disabled

### ISSUE THREAD

**Issue by foxbox-doug** [awslabs/cognito-at-edge#52]
#### What happened:

User attempted to log in with ad block. Error message in log is as follows:

```
User isn't authenticated: TypeError: Cannot read properties of undefined (reading 'flatMap')
```

#### What did you expect to have happen: 

Error message in log indicating that no cookies were found rather than null reference exception.

#### How to reproduce this (as precisely and succinctly as possible): 

Attempt to access protected content using an ad blocker that prevents cookies from being sent.

#### Anything else you think we should know? 

N/A

#### Environment: 
- version of cognito-at-edge being used: 1.3.1
- node version of code base which uses cognito-at-edge: 16.x
- other:

**Comment by foxbox-doug** [comment]
Looks like the actual issue is a timeout. I will extend the timeout to see if that resolves the issue.

**Comment by jeandek** [comment]
Fixed by #53 .
